### PR TITLE
dev, v3.0, v2.1: add DM 1.0 GA upgrade guide

### PR DESCRIPTION
**Pull request by yikeke**
### What is changed, added or deleted? 

add DM 1.0 GA upgrade guide

### What is the related PR or file link(s)? 

To align with docs-cn: https://github.com/pingcap/docs-cn/pull/1832/

### Which version does your change affect? 

dev, v3.0, v2.1

### CODE REVIEW

**Comment by yikeke** [comment]
In the commit [dev, v3.0, v2.1: put dm upgrade doc in the dm folder](https://github.com/pingcap/docs/pull/1511/commits/45c693736568bdd47728c0aaaa2258f32c265cbd), I did the following things: 

- Changed the location of the DM upgrade file from `/dev/how-to/upgrade/data-migration.md` to `/dev/reference/tools/data-migration/upgrade.md` 
- Added the aliases accordingly
- Updated the TOC file

This commit is to keep the DM upgrade file in the same place as other DM files. 

PTAL @lilin90

File: dev/reference/tools/data-migration/upgrade.md [hunk 1]
```diff
@@ -20,6 +21,37 @@ Assuming that V-A, V-B, V-C are three DM versions in chronological order and the
...
+Release Version: v1.0.0-10-geb2889c9
+Git Commit Hash: eb2889c9dcfbff6653be9c8720a32998b4627db9
+Git Branch: release-1.0
+UTC Build Time: 2019-09-06 03:18:48
+Go Version: go version go1.12 linux/amd64
+```
+
+### Main changes
+
+- Try to automatically recover the replication task in common abnormal situations
```
Comment (csuzhangxc) [inline dev/reference/tools/data-migration/upgrade.md]:
```suggestion
- Try to automatically resume the replication task in common abnormal situations
```

File: dev/reference/tools/data-migration/upgrade.md [hunk 2]
```diff
@@ -20,6 +21,37 @@ Assuming that V-A, V-B, V-C are three DM versions in chronological order and the
...
+### Upgrade operation example
+
+1. Download the new version of DM-Ansible, and confirm that there is `dm_version = v1.0.0` in the `inventory.ini` file.
+2. Run `ansible-playbook local_prepare.yml` to download the new DM binary file to the local disk.
+3. Run `ansible-playbook rolling_update.yml` to perform a rolling update for the DM cluster components.
+4. Run `ansible-playbook rolling_update_monitor.yml` to perform a rolling update for the DM monitoring components.
+
+> **Note:**
+>
+> When you upgrade DM to the 1.0 GA version, you must make sure that all DM cluster components (dmctl, DM-master, and DM-worker) are upgraded at the same time.
```
Comment (csuzhangxc) [inline dev/reference/tools/data-migration/upgrade.md]: 这里用 `at the same time` 合适吗？我们中文用 `同时升级` 其实是想表达这次升级操作，需要把所有组件都一起都升级了。

Comment (yikeke) [reply dev/reference/tools/data-migration/upgrade.md]: Updated. I just removed `at the same time` to convey the correct meaning.

**Pull request merged**